upgrading/topics/install_new_version.adoc

```diff
@@ -18,18 +18,48 @@ in the new installation.
 +
 For domain installations, create the empty directory {project_dirref}/domain/deployments.
 +
-NOTE: 
+NOTE:
 Files in the bin directory should not be overwritten by the files from previous versions. Changes should be made manually.
 
 . Copy any custom modules that have been added to the modules directory.
 . Run the applicable upgrade script below.
 
+ifeval::[{project_product}==true]
+To upgrade {project_name} server RPM distribution, complete the following steps:
+
+. Prior to applying the upgrade, handle any open transactions and delete the /var/opt/rh/rh-sso7/lib/keycloak/standalone/data/tx-object-store/ transaction directory.
+. Make sure you have subscribed to proper repositories containing JBOSS EAP and {project_name}.
+
+ subscription-manager repos --enable=jb-eap-7.1-for-rhel-7-server-rpms
+ subscription-manager repos --enable=rh-sso-7.2-for-rhel-7-server-rpms
++
+NOTE:
+ To disable older product repositories for both JBOSS EAP and {project_name} use:
+
+ subscription-manager repos --disable=<OLDER_PRODUCT_REPO>
++
+NOTE:
```

Comment:
This does not render correctly to an info box. The `NOTE: ` has to be followed by text, otherwise it renders just as "NOTE: " text.